gcbmgr

```diff
@@ -293,14 +293,13 @@ submit_it () {
   substitutions="_OFFICIAL_TAG=$OFFICIAL_TAG,_NOMOCK_TAG=$NOMOCK_TAG,_RC_TAG=$RC_TAG"
   substitutions+=",_BUILD_POINT=$BUILD_POINT,_GCP_USER_TAG=$GCP_USER_TAG"
 
-  [[ $COMMAND == stage ]] && substitutions+=",_BUILD_AT_HEAD=$BUILD_AT_HEAD"
+  [[ $COMMAND == stage ]] && substitutions+=",_BUILD_AT_HEAD=$BUILD_AT_HEAD" && substitutions+=",_KUBE_CROSS_VERSION=$KUBE_CROSS_VERSION"
```

Comment:
nit: I'd mildly prefer, but that's only personal style I guess ... definitively no blocker!

```bash
[[ $COMMAND == stage ]] && {
  substitutions+=",_BUILD_AT_HEAD=$BUILD_AT_HEAD" 
  substitutions+=",_KUBE_CROSS_VERSION=$KUBE_CROSS_VERSION"
}
```